unpackaged/config/rd2_post_config/profiles/Admin.profile

```diff
@@ -108,4 +112,14 @@
         <field>npe03__Recurring_Donation__c.npe03__Total__c</field>
         <readable>true</readable>
     </fieldPermissions>
+    <fieldPermissions>
```

Comment:
Since the `RecurringDonationSchedule__c` is protected, it's not possible (or needed) to grant permissions to any fields on this object in unpackaged metadata. These 2 (and the 2 on the Standard profile) will need to be removed.